shell/platform/windows/system_utils.cc

```diff
@@ -54,33 +54,9 @@ std::wstring GetPreferredLanguagesFromMUI() {
 std::vector<std::wstring> GetPreferredLanguages(
```

Comment:
We now have no more use for `GetPreferredLanguagesFromRegistry`. Should we remove the function altogether?

Reply:
Yes please remove this. If needed, we can always use git's history to resurrect the code.